src/org/etools/j1939_84/bus/Bus.java

```diff
@@ -94,4 +94,8 @@ default AutoCloseable log(Function<Packet, String> prefix) throws BusException {
      */
     Packet send(Packet packet) throws BusException;
 
+    default boolean foundImposter() {
```

Comment:
I already added imposterDetected() to trunk based on yesterday's meeting.  I didn't add the default method, to avoid accidentally implementing it on a Bus, like J1939TP, which needs to return "bus.imposterDetected();".